wagtail/documents/forms.py

```diff
@@ -4,11 +4,21 @@
 
 from wagtail.admin import widgets
 from wagtail.admin.forms.collections import (
-    BaseCollectionMemberForm, collection_member_permission_formset_factory)
+    BaseCollectionMemberForm, CollectionChoiceField, collection_member_permission_formset_factory)
+from wagtail.core.models import Collection
 from wagtail.documents.models import Document
 from wagtail.documents.permissions import permission_policy as documents_permission_policy
 
 
+# Callback to allow us to override the default form field for the image file field
```

Comment:
image file field -> collection field

Reply:
Whoops. That was copied from the equivalent function in the images app. And strictly speaking, I should have updated the comment on the image function, too. Both are now fixed in my code.